lms/static/sass/course/layout/_banner_cta.scss

```diff
@@ -0,0 +1,65 @@
+
+@mixin white-button-flat-outline {
```

Comment:
I originally put this back in the day where I had because I thought that sass file was widely available and wanted to encourage more use of it. Ah well. (Not arguing with placement here, since the previous location was **not** widely available.)

Reply:
Yeah, I moved it mainly because nothing else was using it, and locating it in this file was the easiest way to have it consistently available to this code.